src/gui/GUI.java

```diff
@@ -61,6 +61,12 @@ public GUI() {
 		mainMenuFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
 	}
 
+	public GUI(boolean shouldInit) {
```

Comment:
allows us to instantiate the gui without any of the gui elements, to run in a headless environment